### DISCUSSION TITLE
Strongswan IP changes multiple Times a Day

### DISCUSSION THREAD

**Discussion by patsn** [strongswan/strongswan#1107] [Q&A]
Hello everyone.
I am currently using Strongswan on a Server to connect to a VPN.
The given VPN-IP is really important for me.
The Problem here: The VPN-IP changes multiple Times a Day and / or the connection keeps breaking.
This behavior occurs only on one of many Vpn clients.
Since I really don't know anything about Strongswan, I rely on your help.
If you need any conf Settings, let my know which.
Thanks in regards.

**Comment by tobiasbrunner** [comment]
Please see #196.

> The given VPN-IP is really important for me.

What do you mean? And why?

**Reply by patsn** [reply]
I'm running a Web-App inside the VPN and the Vpn IP is the IP the other Users inside the VPN can access my web-app.
And if it changes all the time... All the Users dont know the new IP to reach the App

**Comment by patsn** [comment]
I'm currently logging the Charon. If the IP changes i'll post the result.
Otherwise if anyone knows a solution, let me know.
